sdk/keyvault/azure-keyvault-certificates-apis/azure/keyvault/certificates/_shared/_generated/v2016_10_01/operations/_key_vault_client_operations.pyi

```diff
@@ -0,0 +1,74 @@
+# Stubs for azure.keyvault.certificates._shared._generated.v2016_10_01.operations._key_vault_client_operations (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Optional
+
+class KeyVaultClientOperationsMixin:
+    def create_key(self, vault_base_url: Any, key_name: Any, kty: Any, key_size: Optional[Any] = ..., key_ops: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., curve: Optional[Any] = ..., cls: Optional[Any] = ..., **kwargs: Any): ...
+    def import_key(self, vault_base_url: Any, key_name: Any, key: Any, hsm: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., cls: Optional[Any] = ..., **kwargs: Any): ...
+    def delete_key(self, vault_base_url: Any, key_name: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def update_key(self, vault_base_url: Any, key_name: Any, key_version: Any, key_ops: Optional[Any] = ..., key_attributes: Optional[Any] = ..., tags: Optional[Any] = ..., cls: Optional[Any] = ..., **kwargs: Any): ...
+    def get_key(self, vault_base_url: Any, key_name: Any, key_version: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def get_key_versions(self, vault_base_url: Any, key_name: Any, maxresults: Optional[Any] = ..., cls: Optional[Any] = ..., **kwargs: Any): ...
+    def get_keys(self, vault_base_url: Any, maxresults: Optional[Any] = ..., cls: Optional[Any] = ..., **kwargs: Any): ...
+    def backup_key(self, vault_base_url: Any, key_name: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def restore_key(self, vault_base_url: Any, key_bundle_backup: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def encrypt(self, vault_base_url: Any, key_name: Any, key_version: Any, algorithm: Any, value: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
+    def decrypt(self, vault_base_url: Any, key_name: Any, key_version: Any, algorithm: Any, value: Any, cls: Optional[Any] = ..., **kwargs: Any): ...
```

Comment:
`cls` is a really short-hand-ish way to name this. Is this a Python convention that 'cls' is a callable class? 

From the documentation it sounds like the intention here is to create a request callback. Can that be baked into the name?